tests/helpers/payload-test-manager.ts

```
@@ -0,0 +1,107 @@
+          local: true
+        })
+        
+        this.instances.set(id, instance)
+      } catch (error) {
+        console.error(`Erreur lors de l'initialisation de Payload pour ${id}:`, error)
+        throw error
+      }
+    }
+    
+    return this.instances.get(id)!
+  }
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)